### ISSUE TITLE
Please have a look at this forum Plugin Idea

### ISSUE THREAD

**Issue by viaahmed** [Yeboster/autocomplete-obsidian#2]
#featureRequest

[https://forum.obsidian.md/t/plugin-thesecondbrain-hints/9672?u=viaahmed](https://forum.obsidian.md/t/plugin-thesecondbrain-hints/9672?u=viaahmed)

I wanted to know your views on this Idea, as well as confirm will you add it in your roadmap...

**Comment by Yeboster** [comment]
Hello @viaahmed,
This is a nice suggestion and I quite like it but right now the core features should be the top priorities.
Before I will implement this provider, which needs to scan the whole vault, cache it into a set and on-the-fly show to you the most probable suggestion, I want to create a more simple provider: suggest the most used words into the current file.
This is also the first step towards the whole vault scan, but still, there's a lot to do to make the UX and performance as best as possible.

**Comment by viaahmed** [comment]
@Yeboster I understand, and agree.
I suggest apart from that, isn't it possible that the user can give a list of words ( user made dictionary) in text file, which the plugin feeds in its provider?

**Comment by Yeboster** [comment]
Yes, that feature is in the provider roadmap. I'll create a proper roadmap with providers and core features in the next future. Then we can choose the priority of each feature, based also on user needs.

**Comment by mtomas7** [comment]
Hi, @viaahmed new RC1 has the feature to use a specific file as a provider/dictionary: https://github.com/Yeboster/autocomplete-obsidian/releases/tag/0.8.0.rc1
